controllers/ocicluster_controller.go

```diff
@@ -84,6 +79,9 @@ func (r *OCIClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 		return ctrl.Result{}, err
 	}
+	if len(ociCluster.Spec.Region) > 0 {
+		r.Region = ociCluster.Spec.Region
```

Comment:
this can be blank. Ho do we deal with this scenario? Will we throw an error?(we should not)